modules/nodes-cluster-worker-latency-profiles-using-at-creation.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * scaling-worker-latency-profiles.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nodes-cluster-worker-latency-profiles-using-at-creation_{context}"]
+= Implementing worker latency profiles at cluster creation
+
+The `workerLatencyProfile` must be added to the manifest in the following sequence:
+
+. Create the manifest needed to build the cluster.
+. Edit the `node.config` object to add the name of the profile. You can change the profile at any time as latency increases or decreases.
```

Comment:
In this step you'll want to edit the file at `<cluster-install-dir>/manifests/config-node-default-profile.yaml`

Reply:
I think the second sentence here can be removed and could be replaced with the point below (that you're able to set any profile at cluster creation time Default, MediumUpdateAverageReaction or LowUpdateSlowReaction)

Reply:
Will want to add ` --dir=mycluster` to the manifest command 

Then edit folder path to have `mycluster` first

Reply:
Thank you! I've added another step in the sequence to be clearer. PTAL!

Reply:
Woops. See below. CLosing this conversation.